Run.command

```diff
@@ -1,27 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-py="$( which python )"
-py3="$( which python3 )"
+[[ -x "$( command -v python3 )" ]] && python3="true"
+[[ -x "$( command -v python )" ]] && python="true"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 python_install=""
 script_name="Scripts/updater.py"
 arg="$1"
 
-if [[ "$py3" == "" ]]; then
-	# No python 3
-	if [[ "$py" == "" ]]; then
-		# No python 2 either
-		echo "Python is not installed!"
+if [[ ! "$python3" ]]; then
```

Comment:
equivalent to `[[ "$foo" != "" ]]`. Better var names so we don't need the comments